scipy/stats/_hypotests.py

```diff
@@ -130,3 +131,133 @@ def epps_singleton_2samp(x, y, t=(0.4, 0.8)):
     p = chi2.sf(w, r)
 
     return Epps_Singleton_2sampResult(w, p)
+
+
+def poisson_etest(k1, k2, n1, n2, diff=0, alternative='two-sided'):
+    """
+    Calculate E-test for the mean difference of two samples
+
+    The test requires two samples coming from poisson distribution. This is a
+    right-sided and two-sided test.
+
+    Parameters
+    ----------
+    k1, k2 : int or float
```

Comment:
k1, k2, n1, n2 should be int. (of course, values like 3.0 would still work, but a float like 2.5 does not make sense)

Reply:
Yes, i agree `k1` and `k2` only accept `int` values. Since both are coming from poisson distribution. `k1` and `k2` are observed values of `X1` and `X2` 
<img width="496" alt="Screen Shot 2019-08-05 at 11 45 08" src="https://user-images.githubusercontent.com/19750950/62439582-96784880-b776-11e9-8d72-99af8e72b82b.png">